.gitignore

```diff
@@ -2,4 +2,5 @@ VERSION
 *.log
 *.nupkg
 obj
+
```

Comment:
Remove this one blank line so this file will be a "no change" file and be removed from the PR.  Thanks.

Reply:
Done, but was .history in the original file?